aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import { Box, Grid, Header, Heading, ResponsiveContext } from 'grommet';
+import {
+  AppContainer,
+  PageContainer,
+  PageContainerContext,
+} from '../page-layouts/components';
+import { ContentArea } from '../page-layouts/anatomy/components';
+
+const parentGridColumns = {
+  xsmall: '100%',
+  small: '100%',
+  medium: ['auto', 'auto'],
+  large: ['auto', 'medium'],
+  xlarge: ['auto', 'medium'],
+};
+
+const firstChildGridColumns = {
+  xsmall: '100%',
+  small: '100%',
+  medium: '100%',
+  large: ['auto', 'small'],
+  xlarge: ['auto', 'small'],
+};
```

Comment:
Referencing this as inspiration: https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2231%3A17994

Suggesting a min-max column to promote the concept and awareness.

```suggestion
const firstChildGrid = {
  columns: {
    xsmall: '100%',
    small: '100%',
    medium: '100%',
    large: ['auto', ['small', 'medium']],
    xlarge: ['auto', ['small', 'medium']],
  },
  gap: 'medium'
};
```